test/sockets.test.js

```diff
@@ -1,63 +1,186 @@
 import 'regenerator-runtime/runtime';
 
 import Socket from '../src/sockets';
-import Logger from '../src/logger';
+import { Logger, SOCKET_PING } from 'syft-helpers.js';
 
 import { WebSocket, Server } from 'mock-socket';
 
 global.WebSocket = WebSocket;
 
 const url = 'ws://localhost:8080/';
 
-const onMessage = event => console.log('NEW MESSAGE', event);
-const onOpen = event => console.log('NEW OPEN', event);
-const onClose = event => console.log('NEW CLOSE', event);
-const onError = event => console.log('NEW ERROR', event);
+/**
+ * Creates promise from the first event call
+ * @param emitter Object that emits the event
+ * @param event Event name
+ * @returns {Promise}
+ */
+function makeEventPromise(emitter, event) {
+  let resolver;
+  let promise = new Promise(resolve => resolver = resolve);
+  emitter.on(event, data => resolver(data));
+  return promise;
+}
 
 describe('Sockets', () => {
+  // common test objects
```

Comment:
Comments should always start capitalized.  They should also be fairly descriptive.  To me, I don't think you need any comments in this file at all.  It's a test file - so it should be "strictly business".  Go ahead and remove them unless they're absolutely critical to the understanding of something complex.  :)

Reply:
OK, removed some comments